frontend/src/components/skeletons/Card.tsx

```
@@ -56,8 +57,12 @@ const CardSkeleton: React.FC<CardSkeletonProps> = ({
             <div className="flex flex-col justify-start gap-2">
               {showContributors && (
                 <div className="mt-3 flex w-full flex-wrap items-center gap-2">
+                  // NOSONAR As safe to use index as key - static skeleton items with fixed length
```

Biome (2.1.2)

[error] Wrap comments inside children within braces.

Unsafe fix: Wrap the comments with braces


(lint/suspicious/noCommentText)